### PR TITLE
new tool volIntensityScale

### PR DESCRIPTION
**Pull request by kerautret**
A simple tool to scale vol intensity

### CODE REVIEW

File: volumetric/volIntensityScale.cpp [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+  }
+    //Parse options
+  if (parseOK &&  ! ( vm.count ( "input" ) ) ) {parseOK=false; missingParam ( "--input" );};
+  if (parseOK && ! ( vm.count ( "output" ) ) ) {parseOK=false; missingParam ( "--output" );};
+  
+
+  po::notify ( vm );
+  if (!parseOK || vm.count ( "help" ))
+    {
+      trace.info() << "Scale the image intensity from an input intensity interval [InMin, InMax] into an output interval [OutMin, OutMax]." <<std::endl
```
Comment (dcoeurjo) [inline volumetric/volIntensityScale.cpp]: maybe you should add "Linear rescaling" ?

File: ChangeLog [hunk 2]
```diff
@@ -3,6 +3,8 @@
 * visualisation/:
 	- meshViewer: fix to display mesh with colored faces (.off with colors).
 
+* volumetric/:
+	- volIntensityScale: a simple tool to scale image intensity of volumetric file.
```
Comment (dcoeurjo) [inline ChangeLog]: linear also here ?

File: volumetric/volIntensityScale.cpp (hunk 1 above)
Comment (kerautret) [inline volumetric/volIntensityScale.cpp]: yes it is thank

File: ChangeLog (hunk 2 above)
Comment (kerautret) [inline ChangeLog]: yes

**Comment by kerautret** [comment]
thank done

**Pull request merged**